docs/src/main/paradox/stream/operators/Sink/foldWhile.md

```diff
@@ -0,0 +1,41 @@
+# Sink.foldWhile
+
+Fold over emitted element with a function, where each invocation will get the new element and the result from the previous fold invocation.
+
+@ref[Sink operators](../index.md#sink-operators)
+
+## Signature
+
+@apidoc[Sink.foldWhile](Sink$) { scala="#foldWhile%5BU%2C%20T%5D(zero%3A%20U)(p%3A%20U%20%3D%3E%20Boolean)(f%3A%20(U%2C%20T)%20%3D%3E%20U):org.apache.pekko.stream.scaladsl.Sink[T,scala.concurrent.Future[U]]" java="#foldWhile(java.lang.Object,org.apache.pekko.japi.function.Predicate,org.apache.pekko.japi.function.Function2)" }
```

Comment:
```suggestion
@apidoc[Sink.foldWhile](Sink$) { scala="#foldWhile[U, T](zero: U)(p: U => Boolean)(f: (U, T) => U):org.apache.pekko.stream.scaladsl.Sink[T,scala.concurrent.Future[U]]" java="#foldWhile(java.lang.Object,org.apache.pekko.japi.function.Predicate,org.apache.pekko.japi.function.Function2)" }
```

Reply:
```suggestion
@apidoc[Sink.foldWhile](Sink$) { scala="#foldWhile[U,T](zero:U)(p:U=%3EBoolean)(f:(U,T)=%3EU):org.apache.pekko.stream.scaladsl.Sink[T,scala.concurrent.Future[U]]" java="#foldWhile(java.lang.Object,org.apache.pekko.japi.function.Predicate,org.apache.pekko.japi.function.Function2)" }
```

Reply:
@He-Pin After reducing the spaces, paradox no longer complains. It may help readability.